tls/ttls.h

```diff
@@ -356,6 +356,29 @@ typedef struct {
 	unsigned char			iv_dec[16];
 } TlsXfrm;
 
+typedef struct {
+	/** allowed ciphersuites per version */
+	const int			*ciphersuite_list[4];
+	/** allowed curves	*/
+	const ttls_ecp_group_id		*curve_list;
+	/** allowed signature hashes	*/
+	const int			*sig_hashes;
+
+	/** Own certificate/key list */
+	ttls_key_cert			*key_cert;
+	/** Private section, used for configuration storing. */
+	void				*priv;
+
+	unsigned char			min_minor_ver;
+	unsigned char			max_minor_ver;
+
+	unsigned int endpoint : 1;	/*!< 0: client, 1: server	*/
+	unsigned int authmode : 2;	/*!< TTLS_VERIFY_XXX	*/
+	unsigned int session_tickets : 1; /*!< use session tickets?	*/
+	unsigned int cert_req_ca_list : 1; /*!< enable sending CA list in
+	Certificate Request messages?	*/
+} TlsPeerCfg;
```

Comment:
Please make it in Tempesta coding style instead of copying it as is from mbedTLS old code.